app/styles/components/_key-value-table.scss

```diff
@@ -1,7 +1,6 @@
 .key-value-table {
   th, td {
     padding-bottom: 10px;
-    text-transform: capitalize;
```

Comment:
> Do u know if I still need to go through other areas of the application and change this behavior?

I think so. I did a global search for 'text-transform: capitalize', and it looks like it's in a couple of other style sheets. Feel free to go through em and make a judgement call.